src/domain/index.ts

```diff
@@ -88,20 +71,62 @@ export type Message =
       parsed: string;
     };
 
+export function getTransactionStatusText(status: Transaction["status"]): string {
+  switch (status) {
+    case "initiated":
+      return "Initiated";
+    case "on-hold":
+      return "On Hold";
+    case "completed":
+      return "Completed";
+  }
+}
+
+export function getChainName(chain: Chain): string {
```

Comment:
Same as before, this shouldn't be in the domain. If you need to display it across different parts of the application, it should be a component: `ChainLabel` or `ChainName`.

Reply:
Done. Both utils moved to a new module `utils/labels.ts`.